### PR TITLE
Accessibility Changes

### PR DESCRIPTION
**Pull request by mckent05**
In this pull request, we (@mckent05 and @Meri-MG) have checked and made changes to make my portfolio according to the accessibility requirement. 

The changes made are:
**Page titles**- No issues found
**Image text alternatives**** (alt text) - [included descriptive alt text for img](https://github.com/mckent05/Portfolio_Project/compare/accessibility_branch?expand=1#diff-bd60669fce289cf66b7cc414552c07f192a0bf60d58a59cce3e14d327285b19bL52-R52)
**Text headings** - [h3 tag was changed to h2 according to heading rules](https://github.com/mckent05/Portfolio_Project/compare/accessibility_branch?expand=1#diff-bd60669fce289cf66b7cc414552c07f192a0bf60d58a59cce3e14d327285b19bL96-R96)
**Color contrast** - No issues found
**Resize** - No issues found
**Interaction** - No issues found
**Moving content** - No issues found
**Multimedia** - No issues found
**The basic structure of the page** - No issues found.

### CODE REVIEW

File: mobile.html [hunk 1]
```diff
@@ -29,13 +29,13 @@
                 <div class="group-4">
```
Comment (carlos-ssh) [inline mobile.html:29]: Kindly add your name to your title page:
Screen Shot 2021-09-28 at 9 43 24

A few examples to have a nice title:
- Akinlade Temitope's portifolio
- Akinlade Temitope full-stack webdeveloper

File: mobile.css [hunk 2]
```diff
@@ -2836,9 +2846,9 @@ button:active {
     align-items: center;
     padding: 0;
     position: absolute;
-    width: 90%;
+    width: 100%;
     height: 274px;
-    left: 2%;
+    left: 0;
     top: 343px;
   }
```
Comment (carlos-ssh) [inline mobile.css:2854]: Your hover buttons is missing contrast.
Kindly work on it:
Screen Shot 2021-09-28 at 9 49 10

File: mobile.html [hunk 3]
```diff
@@ -29,13 +29,13 @@
...
                         </div>
                         <div class="wifi">
-                            <img src="icons/Wifi.png" alt="">
+                            <img src="icons/Wifi.png" alt="wifi-icon">
                         </div>
                         <div class="battery">
-                            <img src="icons/Battery.png" alt="">
+                            <img src="icons/Battery.png" alt="battery-icon">
                         </div>
                     </div>
```
Comment (carlos-ssh) [inline mobile.html:41]: - Kindly provide the requested design(font-family and align text at the center):
Screen Shot 2021-09-28 at 9 55 52

**Review by carlos-ssh — CHANGES_REQUESTED** [review]
Hi @mckent05 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: mobile.html (hunk 3 above)
Comment (mckent05) [reply mobile.html:41]: > * Kindly provide the requested design(font-family and align text at the center):
>   Screen Shot 2021-09-28 at 9 55 52

Hey Carlos, please can you throw more light on this requested change.

File: mobile.css (hunk 2 above)
Comment (mckent05) [reply mobile.css:2854]: Can you also offer any advice on what colours to use for my hover buttons as I have used the colours requested on the template?

**Comment by mckent05** [comment]
i want o re-open this PR

**Pull request merged**